src/transformers/modeling_tf_utils.py

```diff
@@ -808,20 +839,29 @@ def prune_heads(self, heads_to_prune):
         """
         raise NotImplementedError
 
-    def save_pretrained(self, save_directory):
+    def save_pretrained(self, save_directory, saved_model=True, version=1):
         """
         Save a model and its configuration file to a directory, so that it can be re-loaded using the
         :func:`~transformers.TFPreTrainedModel.from_pretrained` class method.
 
         Arguments:
             save_directory (:obj:`str`):
                 Directory to which to save. Will be created if it doesn't exist.
+            saved_model (:obj:`bool`, `optional`, defaults to False):
+                If the model has to be saved in saved model format as well or not.
+            version: (:obj:`int`, `optional`, defaults to 1):
+                The version of the saved model.
         """
         if os.path.isfile(save_directory):
             logger.error("Provided path ({}) should be a directory, not a file".format(save_directory))
             return
         os.makedirs(save_directory, exist_ok=True)
 
+        if saved_model:
+            saved_model_dir = os.path.join(save_directory, "saved_model", str(version))
+            self.save(saved_model_dir, include_optimizer=False, signatures=self.serving)
+            logger.info(f"Saved model created in {saved_model_dir}")
```

Comment:
Does TF serving need that exact path? i.e., `dir/saved_model/{version}`?

Reply:
Yes, it does, you can take a look at this basic serving example https://www.tensorflow.org/tfx/serving/serving_basic the exported model has a version number to be able to be used.